virttest/libvirt_cgroup.py

```diff
@@ -275,6 +280,16 @@ def get_virsh_output_dict(self, vm_name=None, virsh_cmd=None):
             logging.error("There is no virsh cmd '%s'", virsh_cmd)
             return None
         result = func(vm_name, ignore_status=True)
+        return self.get_dict_output(result)
+
+    def get_dict_output(self, result):
```

Comment:
This function is to parse the virsh command result into a dict. It is so common usage. So I suggest it should be put into other place, maybe virttest/virsh.py as it is much close to virsh cmd implementation?

Reply:
makes sense, updated.

Reply:
@dzhengfy @Yingshun I have a little concern about moving this function to `virsh`. Those functions in virsh are all about virsh commands which means when we call a virsh.xxx(), xxx is supposed to be an existing virsh command. I guess it feels a little bit weird when a function not including virsh command comes into 'virsh'.

Reply:
@chloerh yeah... I've rechecked the virsh.py, all of functions are related to command(). Thanks for your reminder!
@dzhengfy The module libvirt_cgroup belongs to virttest package, so I wonder if I can keep it in libvirt_cgroup. If we have other requirements in the future, I think we can create a new function under utils_libvirt. How do you think?

Reply:
Accepted.